boards/arm64/intel_socfpga_agilex5_socdk/intel_socfpga_agilex5_socdk_defconfig

```diff
@@ -0,0 +1,51 @@
+# Copyright (c) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+# The Zephyr build from this defconfig is execpted to boot from
+#Intel Arm Trusted Firmware (ATF).
+# Boot Flow is: ATF BL21 -> ATF BL31 -> Zephyr
+
+CONFIG_SOC_SERIES_AGILEX5=y
+CONFIG_SOC_AGILEX5=y
+CONFIG_BOARD_INTEL_SOCFPGA_AGILEX5_SOCDK=y
+
+# Compiler Options
+CONFIG_FORTIFY_SOURCE_RUN_TIME=y
+
+# Arm Features
+CONFIG_ARM_ARCH_TIMER=y
+CONFIG_CACHE_MANAGEMENT=y
+CONFIG_ARMV8_A_NS=y
+
+# Serial Drivers
+CONFIG_SERIAL=y
+CONFIG_UART_INTERRUPT_DRIVEN=y
+CONFIG_UART_NS16550_ACCESS_WORD_ONLY=y
+
+# Enable Console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# Enable Clock Manager
+CONFIG_CLOCK_CONTROL=y
+
+# For Misc Register Map
+CONFIG_SYSCON=y
+
+# Reset Manager
+CONFIG_RESET=y
+
+# Setting higher priority for Intel SOCFPGA reset controller to initialize it before
+# other dependent drivers running at CONFIG_KERNEL_INIT_PRIORITY_DEFAULT which is 40
+CONFIG_RESET_INIT_PRIORITY=35
```

Comment:
Ideally... I'll hack it in my test PR and see if anything else complains, hang in there.

Reply:
Run result: https://github.com/zephyrproject-rtos/zephyr/pull/58388/checks?check_run_id=14941982543, these were all failing already so yes - I think it would be ok to anticipate `RESET_INIT_PRIORITY` to 35 system wide. Maybe do a separate PR for it?

Reply:
So, shall I keep it here as of now as part this commit?

Reply:
can you open a different PR to fix that?

Reply:
Sure, will do. Thank you

Reply:
@carlocaione  @fabiobaltieri  FYI, created a separate PR for this change. Thank you.
https://github.com/zephyrproject-rtos/zephyr/pull/60412